src/fluree/db/flake.cljc

```diff
@@ -497,23 +497,22 @@
   reference: https://www.javamex.com/tutorials/memory/string_memory_usage.shtml"
   [^Flake f]
   (let [o       (o f)
-        o-size  (#?@(:clj (util/case+)
-                     :cljs (condp =)) (dt f)
-                                      const/$xsd:string (* 2 (count o))
-                                      const/$id 30
-                                      const/$xsd:boolean 1
-                                      const/$xsd:long 8
-                                      const/$xsd:int 4
-                                      const/$xsd:short 2
-                                      const/$xsd:double 8
-                                      const/$xsd:float 4
-                                      const/$xsd:byte 1
+        o-size  (condp = (dt f)
```

Comment:
Why did you remove `case+` on clj? I was under the impression that it had dramatically better performance, but I never tested it.